docs/src/algorithms/discretization.md

```diff
@@ -5,3 +5,24 @@ discretize
), Point(0.41871038f0, 0.22148979f0), Point(0.4152017f0, 0.21646582f0), Point(0.4141205f0, 0.21471222f0), Point(0.41227448f0, 0.21589448f0), Point(0.40966362f0, 0.22001258f0), Point(0.40628797f0, 0.22706655f0), Point(0.40214747f0, 0.23705636f0), Point(0.40200475f0, 0.24653101f0), Point(0.40585983f0, 0.25549048f0), Point(0.41371268f0, 0.2639348f0), Point(0.4255633f0, 0.2718639f0), Point(0.4378565f0, 0.28495985f0), Point(0.4505922f0, 0.30322257f0), Point(0.46377045f0, 0.32665208f0), Point(0.47739124f0, 0.35524836f0), Point(0.5046394f0, 0.36442512f0), Point(0.5455148f0, 0.35418236f0), Point(0.60001767f0, 0.32452005f0), Point(0.66814786f0, 0.27543822f0), Point(0.7186763f0, 0.24664374f0), Point(0.75160307f0, 0.23813659f0), Point(0.76692814f0, 0.2499168f0), Point(0.7646515f0, 0.28198436f0), Point(0.7769703f0, 0.29925033f0), Point(0.8038847f0, 0.3017147f0), Point(0.84539455f0, 0.28937748f0), Point(0.9015f0, 0.26223865f0), Point(0.94408435f0, 0.24899776f0), Point(0.9731477f0, 0.24965483f0), Point(0.98869f0, 0.26420987f0), Point(0.9907113f0, 0.29266283f0), Point(0.9849871f0, 0.31338844f0), Point(0.97151726f0, 0.32638666f0), Point(0.950302f0, 0.3316575f0), Point(0.9213412f0, 0.32920095f0), Point(0.8798396f0, 0.34078467f0), Point(0.8257972f0, 0.36640862f0), Point(0.7592141f0, 0.40607283f0), Point(0.6800901f0, 0.4597773f0), Point(0.6450007f0, 0.49104902f0), Point(0.6539457f0, 0.49988794f0), Point(0.7069251f0, 0.48629412f0), Point(0.803939f0, 0.45026752f0), Point(0.877913f0, 0.4226481f0), Point(0.9288472f0, 0.40343583f0), Point(0.9567415f0, 0.39263073f0), Point(0.961596f0, 0.39023277f0), Point(0.9419039f0, 0.40523484f0), Point(0.89766514f0, 0.43763688f0), Point(0.8288798f0, 0.48743892f0), Point(0.7355478f0, 0.55464095f0), Point(0.6655121f0, 0.60063523f0), Point(0.6187727f0, 0.6254217f0), Point(0.5953296f0, 0.62900037f0), Point(0.5951828f0, 0.6113712f0), Point(0.57516366f0, 0.60261106f0), Point(0.53527224f0, 0.6027198f0), Point(0.4755085f0, 0.6116975f0), Point(0.3958725f0, 0.6295441f0), Point(0.33913234f0, 0.6398651f0), Point(0.30528808f0, 0.6426605f0), Point(0.2943397f0, 0.6379303f0), Point(0.30628717f0, 0.6256744f0), Point(0.32149008f0, 0.6093727f0), Point(0.33994842f0, 0.5890249f0), Point(0.36166218f0, 0.5646312f0), Point(0.38663134f0, 0.5361916f0), Point(0.3919681f0, 0.520893f0), Point(0.3776725f0, 0.5187355f0), Point(0.34374446f0, 0.52971905f0), Point(0.29018405f0, 0.5538437f0), Point(0.25439468f0, 0.5678829f0), Point(0.2363764f0, 0.5718367f0), Point(0.23612918f0, 0.56570506f0), Point(0.25365302f0, 0.549488f0), Point(0.2733971f0, 0.5246488f0), Point(0.29536137f0, 0.49118724f0), Point(0.3195459f0, 0.4491035f0), Point(0.34595063f0, 0.39839754f0), Point(0.3647463f0, 0.3590396f0), Point(0.37593287f0, 0.33102974f0), Point(0.37951034f0, 0.31436795f0), Point(0.37547874f0, 0.30905423f0), Point(0.36070493f0, 0.3204269f0), Point(0.33518887f0, 0.348486f0), Point(0.29893062f0, 0.3932315f0), Point(0.25193012f0, 0.45466346f0), Point(0.22926679f0, 0.47329807f0)], Any[])
```

Comment:
You can use the alias `Point2f` instead of `Point{2,Float32}`. Also, you don't have to specify `Point` everywhere: if you specify that the vector contains `Point2f` values, then you can just supply tuple values, which will automatically be converted to the vector type. So, `Point2f[(0.,0.),(1.,1.)]` is equivalent to `[Point2f(0., 0.), Point2f(1., 1.)]`. This should greatly simplify this expression.

Another suggestion may be to use a simpler polygon there (like less than 10 vertices), since the idea is to show the code required for a triangulation using FIST. Then, after a simple example, you can put examples of more complicated polygons, and just show the plots: the polygon before and after triangulation. This would help the user have a more intuitive idea of what the FIST algorithm does, there is no need to show any code. These complicated polygons would be read with the `readpoly` function internally, but only exposed to the documentation through images.

Reply:
Thank you! Working on this.

Reply:
I realized I may not have been clear at the end, when I said "only exposed to the documentation through images" I meant only exposed via plots. We still don't want to manipulate files. So you would probably have one (or more) `@eval` blocks, with a `plot` at the end.

Reply:
@serenity4  When I use:
```julia
...
plot([[1,0], [2,3]])
```
as an example plot I get the following:
![Screenshot from 2021-03-26 18-13-50](https://user-images.githubusercontent.com/14348863/112633238-1e84f780-8e5f-11eb-9790-33188d7dcf8c.png)

I was wondering if it would be okay to use @setup to run the readpoly funcs etc. and just show the plot statement like:
![Screenshot from 2021-03-26 18-19-02](https://user-images.githubusercontent.com/14348863/112633766-c8fd1a80-8e5f-11eb-9e39-e28dab759874.png)

Reply:
Maybe you need to call `display(plot(....))` explicitly then. To be honest I never used plots in my docs so far.
The `plot(mesh)` does not bring anything, so there is no need to show it to the user. I proposed the `@eval` block because the code is not displayed, only the result, which is what we want. As to putting `readpoly` in a `@setup` block, that is the purpose of the block indeed, so that is a good use.

Reply:
I tried display(plot(...)) but it didn't work unfortunately @serenity4  :(

Reply:
@atreyamaj this is supposed to be super simple given that you have similar examples in the GeoStats.jl documentation.  I am still not seeing any update in the docs/Project.toml file so this PR will never show the documentation results. Please add Plots.jl to the docs/Project.toml as I mentioned and try to build the documentation again. Please share the *exact* instructions you are following because we iterating too much to get this done.

As a general comment, please read https://mtlynch.io/code-review-love

We want to be efficient reviewing your code but you need to put more effort into fixing things and thinking on your own.

Reply:
Thank you for the feedback @juliohm 
I have added Plots to docs/Project.toml and added more examples.
Thank you for sharing the resource for code reviews, I am going through it now.